src/components/Icon/index.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'underscore';
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import themeColors from '../../styles/themes/default';
+import variables from '../../styles/variables';
+import * as BrandAssets from './BrandAssets';
+import * as Expensicons from './Expensicons';
+
+const ICONS = _.extend(BrandAssets, Expensicons);
+
+const propTypes = {
+    /* The asset to render. */
+    icon: PropTypes.oneOf(_.values(ICONS)).isRequired,
+
+    /* The width of the icon. */
+    width: PropTypes.number,
+
+    /* The height of the icon. */
+    height: PropTypes.number,
+
+    /* The fill color for the icon */
```

Comment:
Is this a hex value, a color name, or what? Please add a little more detail to the comment to identify what format the string is expected to be.